### DISCUSSION TITLE
How to create pull request in this repository

### DISCUSSION THREAD

**Discussion by ottapav** [qpsolvers/qpbenchmark#59] [General]
I would like to add a subset configuration [#58 ]. I made the changes, I would like to create PR. So I tried to push new branch, but the access was denied. How should I proceed?

Pavel

**Comment by stephane-caron** [comment]
Pull requests are enabled for this repository. The most common way to open a PR on GitHub is to fork the repository, push your changes to your fork, then open the PR to the original repository. See for instance [this tutorial](https://jarv.is/notes/how-to-pull-request-fork-github/).
